.configure

```diff
@@ -22,4 +27,4 @@
   "file_dependencies": [
 
   ]
-}
\ No newline at end of file
+}
```

Comment:
This is my Vim adding a new line at the end of a file by default. [Here's an interesting post on where the convention comes from](https://thoughtbot.com/blog/no-newline-at-end-of-file).